app/assets/stylesheets/style.css.scss

```diff
@@ -99,6 +99,12 @@ fieldset.tagline {
 
 #neighborhood_list img {
   border-radius: 10px;
+  opacity: .67;
+  filter: alpha(opacity=67);
+}
+
+#neighborhood_list a{
+  color: #FF1C8E;
```

Comment:
Color literals like `#FF1C8E` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FF1C8E` should be written as `#ff1c8e`